frontend/src/pages/ChainMemberList.tsx

```diff
@@ -249,15 +250,29 @@ function HostTable(props: {
   }
 
   function onDemote() {
-    if (!selected) return;
```

Comment:
This is a guard keep here to ensure that the user doesn't receive a notification that doesn't work